manifest-lock.overrides.yaml

```
@@ -13,3 +13,8 @@ packages:
     metadata:
       reason: https://github.com/coreos/fedora-coreos-tracker/issues/913
       type: pin
+  shadow-utils:
+    evr: 2:4.8-1.fc35
```

Shouldn't this be

```suggestion
    evr: 2:4.8.1-20.fc35
```

based on https://github.com/coreos/fedora-coreos-tracker/issues/916#issue-962167292?

---

fixed